src/main/kotlin/lotto/Rank.kt

```diff
@@ -0,0 +1,30 @@
+package lotto
+
+enum class Rank(
+    val matchCount: Int,
+    val matchBonus: Boolean,
+    val prizeAmount: Int,
+) {
+    FIRST(6, false, 2_000_000_000),
+    SECOND(5, true, 30_000_000),
+    THIRD(5, false, 1_500_000),
+    FOURTH(4, false, 50_000),
+    FIFTH(3, false, 5_000),
+    NONE(0, false, 0), ;
+
```

Comment:
if `prizeAmount` changes or new rank are added, only need to look at the `Rank` enum class! 💯 👍